tests/test_serializers.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python
+"""
+Tests for the `super-csv` mixins module.
+"""
+
+import json
+from unittest.mock import patch
+
+from django.test import TestCase
+
+from super_csv.serializers import CSVOperation, CSVOperationSerializer
+
+
+class SerializerTestCase(TestCase):
+
+    def setUp(self):
+        super().setUp()
+        self.data = {
+            "total_rows": 3,
+            "processed_rows": 4,
+            "saved_rows": 5
+        }
+
+        self.operation = CSVOperation.record_operation('some name', 'course_id', 'save', json.dumps(self.data))
+
+    def test_get_data_success(self):
+        # Test success get total_rows
+        with patch('simplejson.load') as load:
```

Comment:
Ah, no, I mean you shouldn't have to patch at all, since you're just making the model with those values already. Does that not work?

Reply:
Oh, I thought I had to mock it. Just realize I do not have to do that at all.